src/utils/supports.js

```diff
@@ -39,10 +42,13 @@ function supports(feature) {
 
     const range = features[feature];
     if (!range) return false;
-    return (supportsCache[key] = satisfies(version, range, {
-      includePrerelease: true,
-      loose: true,
-    }));
+    return (supportsCache[key] = (
+      containGTS.test(feature) === containGTS.test(software_name)
```

Comment:
This line is honestly a bit hard to read. I get it, if it's Mastodon, it'll be `false === false` which returns `true` 😬

Could possibly prepend this above instead? e.g.:

```
if (softwareName.toLowerCase() === 'gotosocial' && feature.startsWith('@gotosocial') {
  return satisfies(version, range, {
    includePrerelease: true,
    loose: true,
  });
}
```

Reply:
Hmm, that doesn't quite work, because if the feature is GoToSocial but the server is Mastodon, or vice versa, and if it then falls through to the original `satisfies` check, it's going to give incorrect results. We need some notion of "Does the software that's running on this server match the software this feature applies to?"